runtime/runtime/src/lib.rs

```diff
@@ -321,6 +321,7 @@ impl Runtime {
         state_update: &TrieUpdate,
         receiver_id: &AccountId,
     ) -> Result<Arc<ContractCode>, String> {
+        debug!(target:"runtime", "Calling the contract at account {}", receiver_id);
```

Comment:
We should probably not have too many logs at debug level. Could use `trace!` instead.